po/de.po

```diff
@@ -3127,15 +3127,15 @@ msgstr "Alte Daten abschließen und in vorigen Buchführungsperioden speichern."
 
 #: ../gnucash/gnome/gnc-plugin-basic-commands.c:191
 msgid "_Price Database"
-msgstr "Kurs-Datenbank"
+msgstr "Kurs-Datenbank..."
```

Comment:
What should be do with these ? Should I already commit them ? If so, all the German message strings will get marked as fuzzy the day we add the ellipses in the English strings as well.

Reply:
It depends on how we decide about the pending release and string freeze.